tests/testsuite/pkgid.rs

```diff
@@ -0,0 +1,34 @@
+//! Tests for the `cargo pkgid` command.
+
+use cargo_test_support::project;
+use cargo_test_support::registry::Package;
+
+#[cargo_test]
+fn simple() {
+    Package::new("bar", "0.1.0").publish();
+    let p = project()
+        .file(
+            "Cargo.toml",
+            r#"
+            [package]
+            name = "foo"
+            version = "0.1.0"
+            edition = "2018"
+
+            [dependencies]
+            bar = "0.1.0"
+        "#,
+        )
+        .file("src/main.rs", "fn main() {}")
+        .build();
+
+    p.cargo("build").run();
```

Comment:
```suggestion
    p.cargo("generate-lockfile").run();
```

I think this would make the purpose clearer (`pkgid` just needs a lock file), and would make the test faster (our tests tend to be a little too heavy).

Reply:
I fixed it. Thanks for your review!